src/wilcoxon.jl

```diff
@@ -129,12 +129,15 @@ function pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+ci(x::ExactSignedRankTest, alpha::Real=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+
 
 ## APPROXIMATE SIGNED RANK TEST
 
 immutable ApproximateSignedRankTest{T<:Real} <: HypothesisTest
+    vals::AbstractVector{T} # original values
```

Comment:
Oops. This is unfortunate because the type is abstract. I guess you changed away from `Vector` for a reason so could you share the reason here?